egs/wsj/s5/utils/scoring/prepare_calibration_data.py

```diff
@@ -0,0 +1,151 @@
+#!/bin/env python
```

Comment:
I'm not crazy about the location as utils/scoring/.  The way I'm thinking of this confidence calibration is as a utility that doesn't intrinsically relate to scoring, while 'scoring' refers specifically to the process of getting scores from output (not of producing that output).